src/hyperlight_host/src/func/utils.rs

```diff
@@ -4,6 +4,8 @@
 ///
 /// Usage:
 /// ```rust
+/// use hyperlight_host::func::for_each_tuple;
```

Comment:
was this change intended?

Reply:
Yes, there were two failing docs tests that I fixed. I don't know how on CI they did not fail